lib/convection/model/template/resource/aws_sqs_queue_policy.rb

```diff
@@ -8,9 +9,25 @@ class Resource
         # AWS::SQS::QueuePolicy
         ##
         class SQSQueuePolicy < Resource
+          extend Forwardable
+
           type 'AWS::SQS::QueuePolicy'
-          property :queue, 'Queues', :type => :list
-          property :policy_document, 'PolicyDocument'
+          property :queues, 'Queues'
```

Comment:
Is there a reason for the change from `queue` to `queues`?  If is truly an array, the accumulator is the better approach.

    property :queue, 'Queues', :type => :list


Then in the template it would be this:

    queue 'somequeue'
    queue 'someotherqueue'